### PR TITLE
Added a tab on shipping logs with SELinux

### PR DESCRIPTION
**Pull request by nico-shishkin**
# What changed

https://deploy-preview-1263--logz-docs.netlify.app/shipping/log-sources/rsyslog.html#rsyslog-selinux

----

## Pages to review

- LinkToPage1
- LinkToPage2

## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

**Review by Doron-Bargo — APPROVED** [review]
Looking good

File: _source/logzio_collections/_log-sources/rsyslog.md [hunk 1]
```diff
@@ -123,6 +124,110 @@ If you still don't see your logs, see our [rsyslog troubleshooting guide](https:
...
+
+##### Ensure logs are sent to Logz.io when SELinux is disabled
+
+Temporarily disable SELinux by running this command:
+
+```shell
+$ sudo setenforce 0
+```
+
+Wait a few minutes and check [your Kibana dashboard](https://app.logz.io/#/dashboard/kibana). If you see the logs, we have confirmed that the problem is that we use SELinux and try to send logs over port 5000 TCP.
```
Comment (YajB) [inline _source/logzio_collections/_log-sources/rsyslog.md:190]: Who is "we"? The user? Logz.io? 

This paragraph is unclear.

File: _source/logzio_collections/_log-sources/rsyslog.md [hunk 2]
```diff
@@ -123,6 +124,110 @@ If you still don't see your logs, see our [rsyslog troubleshooting guide](https:
...
+
+
+##### Enable SELinux 
+
+At this point, we need to switch SELinux back on, to ensure the system remains protected.
```
Comment (YajB) [inline _source/logzio_collections/_log-sources/rsyslog.md:195]: I don't understand why the switch to "we"? 
This procedure is performed by the user independently. Logz.io is not involved in the process.

File: _source/logzio_collections/_log-sources/rsyslog.md [hunk 3]
```diff
@@ -123,6 +124,110 @@ If you still don't see your logs, see our [rsyslog troubleshooting guide](https:
...
+
+To do this, run:
+
+```shell
+$ sudo setenforce 1
+```
+
+##### Install the SELinux policy core utilities package
+
+Now we need to add the package `policycoreutils` containing the SELinux command `semanage` that will enable us to modify the policy.
```
Comment (YajB) [inline _source/logzio_collections/_log-sources/rsyslog.md:205]: "we" & "us"?

File: _source/logzio_collections/_log-sources/rsyslog.md [hunk 4]
```diff
@@ -123,6 +124,110 @@ If you still don't see your logs, see our [rsyslog troubleshooting guide](https:
...
+Execute the following command to check if your system sends logs to Logz.io:
+
+```shell
+curl -sLO https://github.com/logzio/logzio-shipper/raw/master/dist/logzio-rsyslog.tar.gz \
+  && tar xzf logzio-rsyslog.tar.gz \
+  && sudo rsyslog/install.sh -t linux -a "<<LOG-SHIPPING-TOKEN>>" -l "<<LISTENER-HOST>>"
+```
+
+
+The above assumes the following defaults:
```
Comment (YajB) [inline _source/logzio_collections/_log-sources/rsyslog.md:166]: Perhaps it would be more logical to list assumptions _before_ the code.

**Review by YajB — APPROVED** [review]

**Pull request merged**